plugins/content/course/model.schema

```diff
@@ -590,6 +590,9 @@
     "themeSettings": {
       "type": "object"
     },
+    "themeVariables": {
```

Comment:
Note: needs adding to framework.

Reply:
I've added a separate issue to this milestone, otherwise this is going to get lost https://github.com/adaptlearning/adapt_authoring/issues/2220